src/template.rs

```diff
@@ -81,9 +82,9 @@ pub struct PageValues {
 }
 
 impl From<Content> for PageValues {
-    fn from(mut c: Content) -> Self {
+    fn from(c: Content) -> Self {
         PageValues {
-            body: c.render_markdown(&None),
+            body: c.body,
```

Comment:
Do we at this point even need `PageValues` anymore, or could we retain `Content` and use that directly instead?

Reply:
Good point. Removed it.